RZViewActions/UIView+RZViewActions.m

```diff
@@ -128,6 +142,17 @@ - (void)_runWithCompletion:(RZViewActionCompletion)completion
 
 @end
 
+#pragma mark - RZViewSpringAction
+
+@implementation RZViewSpringAction
+
+- (void)_runWithCompletion:(RZViewActionCompletion)completion
```

Comment:
@jkaufman Yeah I read those over lunch and was saddened. They suggest prefixing private methods as an alternative, which imo is better than doing nothing.